### ISSUE TITLE
mss620 issues

### ISSUE THREAD

**Issue by AdmiralRaccoon** [vincenzosuraci/hassio_meross_sensor_switch#14]
Hi there,
I hope this project is still alive.
I have 2 mss425e working perfect with this addon.

Now I tried the mss620 (2-socket outdoor plug) and have the following problem: 3 switches were detected, One that switches on both, One for the left and one for the right.
Now the weird part:
After renaming one of them, the other two vanished from HA. After a restart, I saw them again, renamed another switch, and what happens is that the first switch I renamed got overwritten by this one.
To sum up, I can only create one switch, as the older ones always get overwritten.

Here some facts:
HA version: 0.96.1
Meross mss620, Firmware 2.1.6, HW Version 2.0.0

I will post a log as soon as I come back from work in the evening

Thanks,
Philipp

**Comment by AdmiralRaccoon** [comment]
Well, that was too quick.
After the next restart, it seems it´s working now. I have 2 switches for each plug. :-D

**Comment by vincenzosuraci** [comment]
Well, always restart Hassio when making important changes.

**Comment by vincenzosuraci** [comment]
Dear all, I sugget to move on this component : [Alberto Geniola](https://github.com/albertogeniola/meross-homeassistant)
